impeller/tools/build_metal_library.py

```diff
@@ -109,6 +109,10 @@ def main():
           '--std=ios-metal1.2',
           '-mios-version-min=10.0',
       ]
+    # if args.platform == 'ios-simulator':
```

Comment:
Did you mean to leave this in?

Reply:
I uploaded this quickly to kick off a presubmit run before your patch landed so that I can verify that making the log FATAL will cause the scenario test to fail. I'll rebase and remove this when that check is finished.